README.md

```diff
@@ -13,10 +13,8 @@
 [![Build Status](https://travis-ci.org/dune-community/dune-xt-common.svg?branch=master)](https://travis-ci.org/dune-community/dune-xt-common)
 
 dune-xt-common is an eXTensions module for DUNE (http://www.dune-project.org),
```

Comment:
It should say dune-xt instead of dune-xt-common here. While you are at it, you should also update the build status line above to point to the [azure tests](https://dev.azure.com/dune-community/dune-xt/_build?definitionId=10&_a=summary) or maybe just remove it.